### PR TITLE
Map Fragment and Blue Markers

### PR DESCRIPTION
**Pull request by dynamitechetan**
Fixes issue #140 

Changes:
 1. used map fragment instead of map FragmentActivity
 2. Blue markers to match app theme.

**Screenshots for the change:**

### CODE REVIEW

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+            listOfAllImages.add(PathOfImage);
+        }
+        return listOfAllImages;
+    }
+
+    @SuppressWarnings("deprecation")
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState)
+    {
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: move the brace on the same line of function

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
```
Comment (pa1pal) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Remove the unnecessary new lines in the code. These are on lots of places

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        cursor = activity.getContentResolver().query(uri, projection, null,
+                null, null);
+
+        column_index_data = cursor.getColumnIndexOrThrow(MediaStore.MediaColumns.DATA);
+        column_index_folder_name = cursor
+                .getColumnIndexOrThrow(MediaStore.Images.Media.BUCKET_DISPLAY_NAME);
+        while (cursor.moveToNext()) {
+            PathOfImage = cursor.getString(column_index_data);
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Remove this line as well

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+            mSupportMapFragment = SupportMapFragment.newInstance();
+            fragmentTransaction.replace(R.id.mapwhere, mSupportMapFragment).commit();
+        }
+
+        if (mSupportMapFragment != null) {
+            mSupportMapFragment.getMapAsync(new OnMapReadyCallback() {
+                @Override
+                public void onMapReady(GoogleMap googleMap) {
+                    if (googleMap != null) {
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Unwanted blank line.

File: app/src/main/java/vn/mbm/phimp/me/MapFragment.java [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+                            arg0.showInfoWindow();
+                            return true;
+                        }
+
+                    });
+                }
+            });
+
+
```
Comment (Abhi2424shek) [inline app/src/main/java/vn/mbm/phimp/me/MapFragment.java]: Blank lines.

File: app/src/main/res/layout/mapfragment.xml [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+    android:orientation="vertical" android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="250dp"
+        android:layout_weight="2"
+        android:name="com.google.android.gms.maps.SupportMapFragment"
+        android:id="@+id/mapwhere" />
+
+</LinearLayout>
```
Comment (Abhi2424shek) [inline app/src/main/res/layout/mapfragment.xml]: Add a new line at EOF

**Comment by dynamitechetan** [comment]
@pa1pal Check it now. I reverted to red markers. also made a async task to load images. Black screen issue fixed.

**Pull request merged**